### ISSUE TITLE
Fatal Error after update

### ISSUE THREAD

**Issue by DragonEnergy** [woocommerce/woocommerce#27388]
updated to 4.4.0
Fatal error: Declaration of Automattic\WooCommerce\Internal\DependencyManagement\ExtendedContainer::get($id, bool $new = false) must be compatible with League\Container\Container::get($alias, array $args = Array) in ~/html/wp-content/plugins/woocommerce/src/Internal/DependencyManagement/ExtendedContainer.php on line 18

**Comment by Kipperlenny** [comment]
see #27384

**Comment by juliaamosova** [comment]
Hi @DragonEnergy,

Can you please confirm if you are using WP Rocker plugin?

If you use WP Rocket plugin, as @Kipperlenny mentioned, please see issue https://github.com/woocommerce/woocommerce/issues/27384. In short, you'd need to update WP Rocket plugin to version 3.6.4. 

 If you are not using WP Rocket plugin, it would be helpful if you could test your site for a theme and plugins conflict. To do that, you’d need to deactivate all plugins except for WooCommerce and switch the default theme such as [Storefront](https://en-gb.wordpress.org/themes/storefront/). Then test again.

If the issue is resolved with the default theme and all plugins deactivated, it means that one of your plugins or a theme is causing the issue. You will then need to enable it one by one and test every time you do that in order to figure out which plugin is causing the issue. 

Please let us know.

**Comment by Kipperlenny** [comment]
I would guess he is using Jetpack - that has the same error

**Comment by juliaamosova** [comment]
>I would guess he is using Jetpack - that has the same error (but only if loaded after/before WooCommerce - strange things happen there ^^)

@Kipperlenny we identified for the error reported by @DragonEnergy in this thread to be caused by incompatibility with plugins that use PHP League’s Dependency Injection container which is the case of WP Rocket 3.6.3. 

Are you able to reproduce this with WooCommerce + Jetpack only? If so, what version of WooCommerce are you using? 

We opened a separate issue related to the Jetpack autoloader: https://github.com/woocommerce/woocommerce/issues/27390.

**Comment by DragonEnergy** [comment]
@juliaamosova I not using the WP Rocket Plugin

**Comment by DragonEnergy** [comment]
I have since rolledback to WooCommerce 4.3.0 but I have a multitude of REST callback errors in my status reports

egister_rest_route was called incorrectly. The REST API route definition for wc/store/cart is missing the required permission_callback argument. For REST API routes that are intended to be public, use __return_true as the permission callback. 

Notice: register_rest_route was called incorrectly. The REST API route definition for wc/store/cart/add-item is missing the required permission_callback argument. For REST API routes that are intended to be public, use __return_true as the permission callback.

Like 20 of these with different routes: all pointing to functions.php line 5225.

Notice: register_rest_route was called incorrectly. The REST API route definition for wc/store/cart/remove-coupon is missing the required permission_callback argument. For REST API routes that are intended to be public, use __return_true as the permission callback. Please see Debugging in WordPress for more information. (This message was added in version 5.5.0.) in /chroot/home/~/html/wp-includes/functions.php on line 5225

**Comment by DragonEnergy** [comment]
I am also not using JetPack

**Comment by juliaamosova** [comment]
>I have since rolledback to WooCommerce 4.3.0 but I have a multitude of REST callback errors in my status reports

These notices were fixed in WooCommerce 4.3.2 on the WooCommerce Product Blocks side of things (which is a part of WooCommerce Core): https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2926 so you should roll back to 4.3.2 or 4.3.3 instead. 

>I am also not using JetPack

Good to know, thanks! 

Can you please post your System Status Report? The System Status Report is found in your WordPress admin under **WooCommerce > Status**. Please select “Get system report”, then “Copy for support”, and then paste it here.

<hr>

However, it will still be great if you could run themes and plugins conflict if you can to determine which plugin is causing the error you reported in this tread.

**Comment by DragonEnergy** [comment]
I am having issues even copying the status report: This is what I have right now.

2020-08-18T04:14:47+00:00 CRITICAL Uncaught Error: Call to a member function get_items() on boolean in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php:107
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(344): include()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(256): wc_get_template('checkout/thanky...', Array)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(63): WC_Shortcode_Checkout::order_received(24434)
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(72): WC_Shortcode_Checkout::output('')
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(95): WC_Shortcodes::shortcode_wrapper(Array, '')
#5 /chroot/home/ROB/rightonbracket.com/html/wp-inc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php on line 107

2020-08-18T05:19:02+00:00 CRITICAL Uncaught Error: Call to a member function get_items() on boolean in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php:107
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(344): include()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(256): wc_get_template('checkout/thanky...', Array)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(63): WC_Shortcode_Checkout::order_received(14439)
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(72): WC_Shortcode_Checkout::output('')
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(95): WC_Shortcodes::shortcode_wrapper(Array, '')
#5 /chroot/home/ROB/rightonbracket.com/html/wp-inc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php on line 107

2020-08-18T05:29:30+00:00 CRITICAL Uncaught Error: Call to a member function get_items() on boolean in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php:107
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(344): include()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(256): wc_get_template('checkout/thanky...', Array)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(63): WC_Shortcode_Checkout::order_received(3356)
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(72): WC_Shortcode_Checkout::output('')
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(95): WC_Shortcodes::shortcode_wrapper(Array, '')
#5 /chroot/home/ROB/rightonbracket.com/html/wp-incl in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php on line 107

2020-08-18T08:25:02+00:00 CRITICAL Uncaught Error: Call to a member function get_items() on boolean in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php:107
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(344): include()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(256): wc_get_template('checkout/thanky...', Array)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(63): WC_Shortcode_Checkout::order_received(16695)
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(72): WC_Shortcode_Checkout::output('')
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(95): WC_Shortcodes::shortcode_wrapper(Array, '')
#5 /chroot/home/ROB/rightonbracket.com/html/wp-inc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php on line 107

2020-08-18T08:25:37+00:00 CRITICAL Uncaught Error: Call to a member function get_items() on boolean in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php:107
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(344): include()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(256): wc_get_template('checkout/thanky...', Array)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(63): WC_Shortcode_Checkout::order_received(16695)
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(72): WC_Shortcode_Checkout::output('')
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(95): WC_Shortcodes::shortcode_wrapper(Array, '')
#5 /chroot/home/ROB/rightonbracket.com/html/wp-inc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/checkout/thankyou.php on line 107

2020-08-18T21:22:59+00:00 CRITICAL Uncaught Error: Call to undefined function fl_woocommerce_version_check() in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome/woocommerce/content-single-product.php:38
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-includes/template.php(732): require()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(283): load_template('/chroot/home/RO...', false)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(598): wc_get_template_part('content', 'single-product')
#3 /chroot/home/ROB/rightonbracket.com/html/wp-includes/shortcodes.php(343): WC_Shortcodes::product_page(Array, '', 'product_page')
#4 [internal function]: do_shortcode_tag(Array)
#5 /chroot/home/ROB/rightonbracket.com/html/wp-includes/shortcodes.php(218): preg_replace_callback('/\\[(\\[?)(produc...', 'do_shortcode_ta...', '[product_page i...')
#6 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome/woocommerce/content-single-product.php on line 38

2020-08-18T21:23:22+00:00 CRITICAL Uncaught Error: Call to undefined function fl_woocommerce_version_check() in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/content-product.php:40
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-includes/template.php(732): require()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(283): load_template('/chroot/home/RO...', false)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-products.php(660): wc_get_template_part('content', 'product')
#3 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-products.php(100): WC_Shortcode_Products->product_loop()
#4 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-template-loader.php(472): WC_Shortcode_Products->get_content()
#5 /chroot/home/ROB/rightonbracket.com/html/wp-includes/class-wp-hook.php(287): WC in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome-child/woocommerce/content-product.php on line 40

2020-08-18T21:23:29+00:00 CRITICAL Uncaught Error: Call to undefined function fl_woocommerce_version_check() in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome/woocommerce/content-single-product.php:38
Stack trace:
#0 /chroot/home/ROB/rightonbracket.com/html/wp-includes/template.php(732): require()
#1 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/wc-core-functions.php(283): load_template('/chroot/home/RO...', false)
#2 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerce/includes/class-wc-shortcodes.php(598): wc_get_template_part('content', 'single-product')
#3 /chroot/home/ROB/rightonbracket.com/html/wp-includes/shortcodes.php(343): WC_Shortcodes::product_page(Array, '', 'product_page')
#4 [internal function]: do_shortcode_tag(Array)
#5 /chroot/home/ROB/rightonbracket.com/html/wp-includes/shortcodes.php(218): preg_replace_callback('/\\[(\\[?)(produc...', 'do_shortcode_ta...', '[product_page i...')
#6 /chroot/home/ROB/rightonbracket.com/html/wp-content/plugins/woocommerc in /chroot/home/ROB/rightonbracket.com/html/wp-content/themes/flatsome/woocommerce/content-single-product.php on line 38

**Comment by DragonEnergy** [comment]
Had to switch to storefront theme to get this

`
### WordPress Environment ###

WordPress address (URL): https://rightonbracket.com
Site address (URL): https://rightonbracket.com
WC Version: 4.3.0
REST API Version: ✔ 1.0.10
WC Blocks Version: ✔ 2.7.1
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.3.0
Log Directory Writable: ✔
WP Version: 5.5
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: Apache/2.4.41 (CentOS)
PHP Version: 7.2.33
PHP Post Max Size: 2 GB
PHP Time Limit: 0
PHP Max Input Vars: 1000
cURL Version: 7.29.0
NSS/3.44

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.2.33-MariaDB
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.3.0
WC Database Prefix: wpyg_
Total Database Size: 118.32MB
Database Data Size: 91.62MB
Database Index Size: 26.70MB
wpyg_woocommerce_sessions: Data: 3.02MB + Index: 0.13MB + Engine InnoDB
wpyg_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_woocommerce_order_items: Data: 0.25MB + Index: 0.09MB + Engine InnoDB
wpyg_woocommerce_order_itemmeta: Data: 2.52MB + Index: 3.02MB + Engine InnoDB
wpyg_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_actionscheduler_actions: Data: 2.02MB + Index: 0.53MB + Engine InnoDB
wpyg_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_actionscheduler_logs: Data: 1.02MB + Index: 0.30MB + Engine InnoDB
wpyg_amazon_accounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_btg: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_amazon_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_amazon_feeds: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_feed_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_feed_tpl_data: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wpyg_amazon_feed_tpl_values: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_amazon_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_listings: Data: 0.02MB + Index: 0.19MB + Engine InnoDB
wpyg_amazon_log: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_amazon_markets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_orders: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_amazon_payment: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_profiles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_reports: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_shipping: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_amazon_stock_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ced_umb_ebayfileTracker: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ced_umb_ebayprofiles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ced_umb_ebay_description_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_comments: Data: 2.28MB + Index: 0.84MB + Engine InnoDB
wpyg_ebay_accounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_auctions: Data: 0.02MB + Index: 0.14MB + Engine InnoDB
wpyg_ebay_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_ebay_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_log: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_ebay_messages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_orders: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wpyg_ebay_payment: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_profiles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_shipping: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_sites: Data: 0.03MB + Index: 0.00MB + Engine InnoDB
wpyg_ebay_store_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_ebay_transactions: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wpyg_ewwwio_images: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wpyg_ewwwio_queue: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_failed_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_fm_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_access: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2018_01: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2019_01: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_01: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_04: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_05: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_06: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_07: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_blob_2020_08: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_archive_numeric_2018_01: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2019_01: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_01: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_04: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_05: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_06: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_07: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_archive_numeric_2020_08: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_brute_force_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_goal: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_locks: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_logger_message: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_log_action: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_log_conversion: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_log_conversion_item: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_log_link_visit_action: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_log_profiling: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_log_visit: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_matomo_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_plugin_setting: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_privacy_logdata_anonymizations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_report: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_report_subscriptions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_segment: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_sequence: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_session: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_site: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_site_setting: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_site_url: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_tagmanager_container: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_tagmanager_container_release: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_tagmanager_container_version: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_matomo_tagmanager_tag: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_tagmanager_trigger: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_tagmanager_variable: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_tracking_failure: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_twofactor_recovery_code: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_user: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_matomo_user_dashboard: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_matomo_user_language: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_options: Data: 6.41MB + Index: 0.16MB + Engine InnoDB
wpyg_pmxe_exports: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxe_google_cats: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxe_posts: Data: 0.16MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxe_templates: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxi_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxi_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxi_imports: Data: 0.16MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxi_posts: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wpyg_pmxi_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_postmeta: Data: 19.39MB + Index: 9.67MB + Engine InnoDB
wpyg_posts: Data: 13.50MB + Index: 0.86MB + Engine InnoDB
wpyg_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_redirection_404: Data: 0.17MB + Index: 0.11MB + Engine InnoDB
wpyg_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_redirection_items: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wpyg_redirection_logs: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wpyg_sendpress_autoresponders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_connections: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_customfields: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_list_subscribers: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_sendpress_queue: Data: 0.02MB + Index: 0.16MB + Engine InnoDB
wpyg_sendpress_schedules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_subscribers: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_sendpress_subscribers_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_sendpress_subscribers_status: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_subscribers_tracker: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_sendpress_subscribers_url: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_sendpress_url: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_smush_dir_images: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_social_users: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_termmeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wpyg_terms: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
wpyg_term_relationships: Data: 0.08MB + Index: 0.05MB + Engine InnoDB
wpyg_term_taxonomy: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wpyg_usermeta: Data: 4.50MB + Index: 5.02MB + Engine InnoDB
wpyg_users: Data: 0.44MB + Index: 0.42MB + Engine InnoDB
wpyg_w3tc_cdn_pathmap: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_w3tc_cdn_queue: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wc_customer_lookup: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wpyg_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_wc_order_product_lookup: Data: 0.20MB + Index: 0.25MB + Engine InnoDB
wpyg_wc_order_stats: Data: 0.14MB + Index: 0.17MB + Engine InnoDB
wpyg_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_wc_product_meta_lookup: Data: 0.06MB + Index: 0.13MB + Engine InnoDB
wpyg_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wfBlockedIPLog: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfBlocks7: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_wfConfig: Data: 1.31MB + Index: 0.00MB + Engine InnoDB
wpyg_wfCrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfFileChanges: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfFileMods: Data: 17.55MB + Index: 0.00MB + Engine InnoDB
wpyg_wfHits: Data: 1.02MB + Index: 0.05MB + Engine InnoDB
wpyg_wfHoover: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wfIssues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_wfKnownFileList: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
wpyg_wfLiveTrafficHuman: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wfLocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfLogins: Data: 0.28MB + Index: 0.11MB + Engine InnoDB
wpyg_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfNotifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfPendingIssues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wpyg_wfReverseCache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wfSNIPCache: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_wfStatus: Data: 0.25MB + Index: 0.16MB + Engine InnoDB
wpyg_wfTrafficRates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wpyg_yoast_indexable: Data: 2.52MB + Index: 0.97MB + Engine InnoDB
wpyg_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wpyg_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wpyg_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wpyg_yoast_seo_links: Data: 0.13MB + Index: 0.05MB + Engine InnoDB
wpyg_yoast_seo_meta: Data: 1.47MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

attachment: 1447
blocks: 9
braintree_log: 3
custom_css: 1
jetpack_migration: 2
jp_img_sitemap: 1
jp_sitemap: 1
jp_sitemap_master: 1
nav_menu_item: 206
oembed_cache: 7
page: 29
post: 78
product: 30
product_variation: 250
revision: 1312
sendpress_list: 1
shop_coupon: 29
shop_order: 1394
shop_order_refund: 35
sp_template: 2
sptemplates: 2
wpcf7_contact_form: 9

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (28) ###

All-in-One WP Migration: by ServMask – 7.26
Better Search Replace: by Delicious Brains – 1.3.3
Classic Editor: by WordPress Contributors – 1.6
Cloudinary: by Cloudinary Ltd.
XWP – 2.1.9

Contact Form 7 Captcha: by 247wd – 0.0.7
Contact Form 7: by Takayuki Miyoshi – 5.2.1
Login No Captcha reCAPTCHA (Google): by Robert Peake – 1.6.10
Nextend Social Login: by Nextendweb – 3.0.24
PDF Embedder: by Lever Technology LLC – 4.6
PixelYourSite: by PixelYourSite – 7.1.11
Privy Website Widget: by Privy Inc. – 2.0.11
Really Simple SSL: by Really Simple Plugins – 3.3.4
Redirection: by John Godley – 4.8
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.3
Lazy Load - Optimize Images: by WP Rocket – 2.3.3
Top Bar: by WP Darko – 3.0
Tracking Code Manager: by IntellyWP – 1.12.3
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.16.26

W3 Total Cache: by BoldGrid – 0.14.4
WooCommerce Stripe Gateway: by WooCommerce – 4.5.1
WooCommerce: by Automattic – 4.3.0 (update to version 4.4.0 is available)
WooSidebars: by WooCommerce – 1.4.5
Wordfence Security: by Wordfence – 7.4.10
Yoast SEO: by Team Yoast – 14.8
WP All Export: by Soflyy – 1.2.7
LiveChat: by LiveChat – 4.3.3
WP Mail SMTP: by WPForms – 2.2.1
WPS Hide Login: by WPServeur
NicolasKulka
tabrisrp – 1.5.6


### Inactive Plugins (8) ###

Matomo Analytics - Ethical Stats. Powerful Insights.: by Matomo – 1.2.0 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Admin: by WooCommerce – 1.3.2 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Amazon Pay: by WooCommerce – 1.12.2 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Dropshipping: by WooCommerce – 1.2.18 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Extended Coupon Features FREE: by Soft79 – 3.1.4 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Google Ads Conversion Tracking: by Wolf+Bär Agency – 1.6.8 – Installed version not tested with active version of WooCommerce 4.3.0
XPS Ship Integration: by XPS Ship – 1.0.93 – Installed version not tested with active version of WooCommerce 4.3.0
YITH WooCommerce Wishlist: by YITH – 3.0.13

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: object-cache.php

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #5 - /heavy-duty-support-brackets/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: #179 - /terms-conditions/

### Theme ###

Name: Storefront
Version: 2.6.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 1,880
Oldest: 2020-07-18 22:29:47 +0000
Newest: 2020-08-18 22:06:45 +0000

Pending: 3
Oldest: 2020-08-18 22:23:32 +0000
Newest: 2020-08-19 18:42:56 +0000

`

**Comment by DragonEnergy** [comment]
upgraded to WooCommerce 4.3.3 most of the errors went away

**Comment by peterfabian** [comment]
Thanks for opening the issue. This is a duplicate of the problem reported in https://github.com/woocommerce/woocommerce/issues/27384 so I'm going to close this one. Please follow the updates on the other issue. The plugin we're in conflict in this case is Lazy Load - Optimize Images: by WP Rocket – 2.3.3.
